config/routes.rb

```diff
@@ -84,6 +84,7 @@
         get 'status_help'
         get 'help'
         get ':time/log', action: :show, as: 'show'
+        delete '', action: :delete
```

Comment:
Could this just be `:destroy` on the containing resource?

Reply:
Was already working on that, ha! New commit just pushed.

Reply:
No, I see the difficulty, a delete would imply deleting one with an ID, but this is the bulk job for the apo.